src/test/java/com/bluesource/projects/Show_Rate_on_Project_Show_Page_Verify.java

```diff
@@ -0,0 +1,76 @@
+package com.bluesource.projects;
+
+import com.orasi.bluesource.Accounts;
+import com.orasi.bluesource.Header;
+import com.orasi.bluesource.LoginPage;
+import com.orasi.utils.Constants;
+import com.orasi.utils.TestReporter;
+import com.orasi.utils.dataProviders.ExcelDataProvider;
+import com.orasi.web.WebBaseTest;
+import org.testng.ITestContext;
+import org.testng.annotations.*;
+
+public class Show_Rate_on_Project_Show_Page_Verify extends WebBaseTest {
+	//*****************
+	// Data Provider
+	//*****************
+	@DataProvider(name = "show_project_rate_verify", parallel = true)
+	public Object[][]scenarions(){
+		return new ExcelDataProvider(Constants.EXCEL_SHEETS + "/ShowRateOnProjectVerify.xlsx","Sheet1").getTestData();
+	}
+
+	@BeforeMethod
+	@Parameters({ "runLocation", "browserUnderTest", "browserVersion",
+			"operatingSystem", "environment" })
+	public void setup(@Optional String runLocation, String browserUnderTest,
+					  String browserVersion, String operatingSystem, String environment) {
+		setApplicationUnderTest("BLUESOURCE");
+		setBrowserUnderTest(browserUnderTest);
+		setBrowserVersion(browserVersion);
+		setOperatingSystem(operatingSystem);
+		setRunLocation(runLocation);
+		setEnvironment(environment);
+		setThreadDriver(true);
+		testStart("Show_Rate_on_Project_Show_Page_Verify");
+	}
+
+	@AfterMethod
+	public void close(ITestContext testResults){
+		endTest("TestAlert", testResults);
+	}
+
+	@Test(dataProvider = "show_project_rate_verify")
+	public void Show_Rate_on_Project_Show_Page_Verify(String scenario, String account, String project, String role)
+	{
```

Comment:
The only comment on I would make on this test method, is you might want to add some assertion statements to verify you've transitioned to right page, or page has displayed, etc.  That way you'll get more useful reporting when it does fail.  So adding an assertion that logged in, accounts page displays, project page displays, etc